bitnami/moodle/templates/deployment.yaml

```diff
@@ -309,6 +313,8 @@ spec:
             initialDelaySeconds: 5
             timeoutSeconds: 1
           resources: {{- toYaml .Values.metrics.resources | nindent 12 }}
+          {{- else if ne .Values.metrics.resourcesPreset "none" }}
+          resources: {{- include "common.resources.preset" (dict "type" .Values.metrics.resourcesPreset) | nindent 12 }}
```

Comment:
Missing `{{- if .Values.metrics.resources }}` in line 314, and `{{- end }}` in line 318.